### ISSUE TITLE
Create About Us page

### ISSUE THREAD

**Issue by Algon-33** [StampyAI/stampy-ui#864]
Multiple people have asked for an about us page. This should be linked to consistently at the bottom of the page. Probably, we should put Rob's profile there as the project founder, and have some text about the rest of the team. I'm happy to write the article, but I can't change the UI of the site.

**Comment by Algon-33** [comment]
To clarify: What I'd like is two bits of text to be added to the bottom of the page. One is just the words "About us" with a hyperlink to https://aisafety.info/questions/NLZQ/What-is-this-site-about. The other is just the text "Copyright Aisafety.info is an Ashgro Inc. Project. Ashgro Inc (EIN: 88-4232889) is a 501(c)(3) Public Charity incorporated in Delaware."

**Comment by LeMurphant** [comment]
At the bottom, there is currently 

`[© AISafety.info, 2022—2025](https://coda.io/d/AI-Safety-Info-Dashboard_dfau7sl2hmG/Copyright_su79L#_luPMa)`

Do you want to keep it as is?

![Image](https://github.com/user-attachments/assets/15447399-39b1-4c3b-a37b-0f8afc1e6d3f)

I'd add a fifth column, but this would mess out the layout. Should "About us" be elsewhere?

**Comment by Algon-33** [comment]
No, this is fine. Except change "Copyright AIsafety.info is an Ashgro ..." to "AIsafety.info is an Ashgro ..."

**Comment by LeMurphant** [comment]
Added the footer. If there are updates to be done, we can reopen the ticket.
